kolibri/core/discovery/utils/filesystem/base.py

```diff
@@ -0,0 +1,78 @@
+import logging
```

Comment:
This file is the interface most other apps would use; internally, it defers to windows.py or posix.py as appropriate.